src/plugins/workspace/server/integration_tests/duplicate.test.ts

```diff
@@ -0,0 +1,306 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import * as exportMock from '../../../../core/server/saved_objects/export';
+import supertest from 'supertest';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { SavedObjectsErrorHelpers } from '../../../../core/server/saved_objects';
+import { UnwrapPromise } from '@osd/utility-types';
+import { loggingSystemMock, savedObjectsClientMock } from '../../../../core/server/mocks';
+import { setupServer } from '../../../../core/server/test_utils';
+// eslint-disable-next-line @osd/eslint/no-restricted-paths
+import { workspaceClientMock } from '../../public/workspace_client.mock';
```

Comment:
nit: no-restricted-paths

use a common directory. move workspaceClientMock to common directory and update all the import statement